studio/src/mas-fragment.js

```diff
@@ -0,0 +1,89 @@
+import { LitElement, html, nothing } from 'lit';
+import StoreController from './reactivity/store-controller.js';
+import Store from './store.js';
+import './mas-fragment-render.js';
+import './mas-fragment-table.js';
+import { reactiveStore } from './reactivity/reactive-store.js';
+import { editFragment } from './editors/merch-card-editor.js';
+
+const tooltipTimeout = reactiveStore(null);
+
+class MasFragment extends LitElement {
+    static properties = {
+        store: { type: Object, attribute: false },
+        view: { type: String, attribute: true }, // 'render' | 'table'
+    };
+
+    createRenderRoot() {
+        return this;
+    }
+
+    selecting = new StoreController(this, Store.selecting);
+    selection = new StoreController(this, Store.selection);
```

Comment:
dupe

Reply:
It's not, I'm subscribing to 2 different stores there 😄 One that specifies whether we're in selection mode, one that has the actual selection (an array of fragment ids)
To be honest, it might be better to set selection at fragment level (and I noticed on the main branch that there was something like that), but again, this is something we can come back to.

Reply:
that confusion is talkative: we should may be find another name that is less confusing for `selecting`